recipes/tree-sitter-langs

```diff
@@ -1,4 +1,4 @@
-(tree-sitter-langs :repo "ubolonton/emacs-tree-sitter"
+(tree-sitter-langs :repo "ubolonton/tree-sitter-langs"
                    :fetcher github
-                   :files ("langs/*.el"
-                           "langs/queries"))
+                   :files ("*.el"
+                           "queries"))
```

Comment:
```suggestion
                   :files (:defaults
                           "queries"))
```
Thanks - I wonder if using the `:defaults` would be helpful here, since it will give your package a bit of room to grow (e.g. to include tests, etc.) without another PR?